stdlib/3/multiprocessing/connection.pyi

```diff
@@ -1,36 +1,61 @@
-from typing import Any, Iterable, List, Optional, Tuple, Type, Union
 import socket
 import types
+from multiprocessing.connection import Listener
+from typing import Any, Iterable, List, Optional, Tuple, Type, Union
 
 # https://docs.python.org/3/library/multiprocessing.html#address-formats
 _Address = Union[str, Tuple[str, int]]
 
+class _ConnectionBase:
+    @property
+    def closed(self) -> bool: ...
+    @property
+    def readable(self) -> bool: ...
+    @property
+    def writable(self) -> bool: ...
+    def fileno(self) -> int: ...
+    def close(self) -> None: ...
+    def send_bytes(
+        self, buf: bytes, offset: int = ..., size: Optional[int] = ...
+    ) -> None: ...
+    def send(self, obj: Any) -> None: ...
+    def recv_bytes(self, maxlength: Optional[int] = ...) -> bytes: ...
+    def recv_bytes_into(self, buf: Any, offset: int = ...) -> int: ...
+    def recv(self) -> Any: ...
+    def poll(self, timeout: Optional[float] = ...) -> bool: ...
+
+class Connection(_ConnectionBase): ...
+class PipeConnection(_ConnectionBase): ...
```

Comment:
You tried to use `if _winapi:` here. While that does not work, `if sys.platform == "win32":` should.